readme.md

```diff
@@ -40,6 +40,7 @@ Some good apps written with Electron.
 
 - [electron-boilerplate](https://github.com/sindresorhus/electron-boilerplate) - Boilerplate to kickstart creating an app.
 - [generator-electron](https://github.com/sindresorhus/generator-electron) - Scaffold out an app boilerplate.
+- [Electron-React-Boilerplate](https://github.com/airtoxin/Electron-React-Boilerplate) - Boilerplate to create Electron app with React.js.
```

Comment:
to create Electron app -> to create an app

According to the [contribution gudelines](https://github.com/sindresorhus/awesome-electron/blob/ee1614289d523b2c55df84cdfea2a6fa2171c511/contributing.md):

> Don't mention `Electron` in the description as it's implied.